demo/api/src/db/fixtures/fixtures.module.ts

```diff
@@ -1,16 +1,65 @@
 import { Module } from "@nestjs/common";
 import { ConfigModule } from "@src/config/config.module";
 import { FixturesConsole } from "@src/db/fixtures/fixtures.console";
-import { ImageFixtureService } from "@src/db/fixtures/generators/image-fixture.service";
 import { LinksModule } from "@src/links/links.module";
 import { PagesModule } from "@src/pages/pages.module";
 import { ConsoleModule } from "nestjs-console";
 
+import { AnchorBlockFixtureService } from "./generators/blocks/anchor.fixture";
```

Comment:
Should the files be named after the services? For example `anchor-block-fixture.service.ts`?

Reply:
I think so yes, I renamed the files